spec/Body/Body.vspec

```diff
@@ -161,11 +161,57 @@ Windshield.WasherFluid.Level:
 ##
 Lights:
   type: branch
-  description: All lights.
+  instances: ["Beam", "Running", "Backup", "Parking", "Brake", "Fog", "Hazard", "Indicator"]
+  description: Exterior lights.
 
-# Include lights specification and attach it to the Lights branch
-#include ExteriorLights.vspec Lights
+Lights.Beam:
+  type: branch
+  instances: ["Low","High"]
+  description: Beam lights.
+#include StaticLights.vspec Lights.Beam
+
+Lights.Running:
+  type: branch
+  description: Running lights.
+#include StaticLights.vspec Lights.Running
+
+Lights.Backup:
+  type: branch
+  description: Backup lights.
+#include StaticLights.vspec Lights.Backup
+
+Lights.Parking:
+  type: branch
+  description: Parking lights.
+#include StaticLights.vspec Lights.Parking
+
+Lights.Fog:
+  type: branch
+  instances: ["Rear","Front"]
+  description: Fog lights.
+#include StaticLights.vspec Lights.Fog
+
+Lights.LicensePlate:
+  type: branch
+  instances: ["Rear","Front"]
+  description: License plate lights.
+#include StaticLights.vspec Lights.LicensePlate
+
+Lights.Brake:
+  type: branch
+  instance: ["Left","Right","Top"]
```

Comment:
What is the use case of having separate instances here? In which cases are they controlled separately?

Reply:
Several instances of the same type of light might have different statuses at some point. For example, if one of the brake lights is defective, it would differ from the others.

Reply:
But that is true also for others like `Lights.Fog`, the left front fog light might be ok, but the right front fog light might be broken. For the others (except turning indicators) we do not separate between left and right.

Reply:
By the way, a syntax error, it shall be `instances:`, not `instance:`

Reply:
And the lack of `description:` on this node causes the binary exporter to crash

Reply:
Meeting notes:

- Possible use case example: Left brake light not used if left turn indicator is active at the same time. If so, there is a business case to control them separately.
- Possibly start without separation, then add separation later if needed

Adnan to check PR status with author